cmd/main.go

```diff
@@ -223,7 +223,6 @@ func main() {
 	var osImagesArray models.OsImages
 	if Options.OsImages == "" {
 		log.Fatal("OS_IMAGES list is empty")
-		osImagesArray = models.OsImages{}
 	} else {
```

Comment:
Nice catch for the osImagesArray line!

Very minor: else block not needed since the if block above will exit if the OsImages is empty